doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,10 @@
 
 <h3>New features since last release</h3>
 
+* Added `show_wire_labels` option to `tape_text` and `tape_mpl`, which hides wire labels when set to `False`.
```

Comment:
```suggestion
* Added `show_wire_labels` option to `draw` and `draw_mpl`, which hides wire labels when set to `False`.
```

Reply:
If it's just for that single change, can we squeeze that in elsewhere conveniently? Rerunning CI for it might be worth skipping :D